### DISCUSSION TITLE
Is it possible to api call for specific encoding links on a media file?

### DISCUSSION THREAD

**Discussion by homegrownhrbs** [mediacms-io/mediacms#1080] [Q&A]
I'm hosting and managing all me media with mediacms.  I'm building a very dumb multi viewer video player and takes media links and allows the user to swap around between the video files as separate viewpoints of the same scene.

My question is, is it possible to make an authorized api call to retrieve the links to the encodings of a specific video file? I can find them manually via the Django backend but automating it would be very time saving for me.

**Comment by mgogoulos** [comment]
Hi, I believe this is what you are asking for: `/api/v1/media/{media_id}`

Eg for this video: https://demo.mediacms.io/view?m=K6NHGGzYB , you can see the encodings at 
https://demo.mediacms.io/api/v1/media/K6NHGGzYB

**Reply by homegrownhrbs** [reply]
you are awesome! thank you sir!
